master/pom.xml

```diff
@@ -1789,6 +1789,16 @@
 
     <plugins>
       <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-eclipse-plugin</artifactId>
+        <version>2.10</version>
+        <!-- https://issues.apache.org/jira/browse/MECLIPSE-726 -->
+        <configuration>
+          <useProjectReferences>true</useProjectReferences>
```

Comment:
Yeah, most likely the fix is the <workspace> that's there. I could probably specify -Declipse.workspace as you said and get the same result. I feel like it's getting messed up for me because I'm trying to import a tool into a different workspace and it's generating the wrong projects.